DataFormats/PortableTestObjects/BuildFile.xml

```diff
@@ -4,7 +4,7 @@
 <use name="DataFormats/Portable"/>
 <use name="DataFormats/SoATemplate"/>
 <use name="HeterogeneousCore/AlpakaInterface"/>
-<flags ALPAKA_BACKENDS="1"/>
+<flags ALPAKA_BACKENDS="cuda rocm"/>
```

Comment:
Assuming this approach of having the "serial" data formats in the main (non-alpaka) directory becomes the preferred one, we should find a way to automate this.

Reply:
I see @smuzaffar opened an issue https://github.com/cms-sw/cmssw-config/issues/93 to improve the syntax, so how about we proceed now with this approach and invert it later on? (I'd expect the improvement to be available before we would be adding a new backend)

Reply:
The next "backend" we could add (even if more for testing purpose than for use in production) would probably be the CPU block-parallel one based on TBB.
If we add that, `<flags ALPAKA_BACKENDS="cuda rocm"/>` would still be the correct choice for any `DataFormats`.

So, yes, I agree we can move ahead with this approach, and improve things later.